src/Compilers/VisualBasic/Portable/Compilation/VisualBasicCompilation.vb

```diff
@@ -2727,6 +2727,14 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             Throw New NotSupportedException(VBResources.ThereAreNoPointerTypesInVB)
         End Function
 
+        Protected Overrides Function CommonCreateFunctionPointerTypeSymbol(
```

Comment:
>CommonCreateFunctionPointerTypeSymbol [](start = 37, length = 37)

Please test this method. Perhaps simply update `CompilationAPITests.CompilationStackOverflow()`.

Reply:
I'm going to rename that test...

---
In reply to: [431405059](https://github.com/dotnet/roslyn/pull/44436#discussion_r431405059) [](ancestors = 431405059)